geonode_mapstore_client/context_processors.py

```diff
@@ -31,6 +31,7 @@ def resource_urls(request):
         'DATASET_MAX_UPLOAD_SIZE': get_max_upload_size("dataset_upload_size"),
         'DOCUMENT_MAX_UPLOAD_SIZE': get_max_upload_size("document_upload_size"),
         'DEFAULT_LAYER_FORMAT': getattr(settings, "DEFAULT_LAYER_FORMAT", 'image/png'),
+        'DEFAULT_MAX_PARALLEL_UPLOADS': get_max_upload_parallelism_limit("default_max_parallel_uploads"),
```

Comment:
@marthamareal let's remove the "default" prefix. It's confusing, since we have a default value inside the settings, which is only used to populate the DB value at install time.